### ISSUE TITLE
bug(web): fail to call sentry-cli in build

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman#6195]
```
[05:32:20][Step 2/6] > keyman@15.0.190 sentry-cli
[05:32:20][Step 2/6] > sentry-cli "releases" "files" "release-15.0.190-alpha" "upload-sourcemaps" "--strip-common-prefix" "release/embedded" "--rewrite" "--ext" "js" "--ext" "map" "--ext" "ts"
[05:32:20][Step 2/6] 
[05:32:21][Step 2/6] error: failed to invoke sentry-cli
[05:32:21][Step 2/6] Error: spawn C:\BuildAgent\work\78a54344b259d3bb\keyman\web\node_modules\@sentry\cli\sentry-cli.exe ENOENT
[05:32:22][Step 2/6]     at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
[05:32:22][Step 2/6]     at onErrorNT (node:internal/child_process:477:16)
[05:32:22][Step 2/6]     at processTicksAndRejections (node:internal/process/task_queues:83:21)
[05:32:22][Step 2/6] Upload successful.
```
https://build.palaso.org/viewLog.html?buildId=305641&buildTypeId=KeymanAndroid_Build&tab=buildLog

However, it doesn't fail the build.

**Comment by darcywong00** [comment]
Moving to web.
The error in the log is coming from this line

https://github.com/keymanapp/keyman/blob/93af2a3d84489e15f514c553546805c2fe90fd5e/web/source/build.sh#L442

I guess we need to catch errors coming from `npm run sentry-cli...`

**Comment by mcdurdin** [comment]
I would like to resolve this after I have cleaned up the build scripts, as it will hopefully be much clearer then.

Depends on #6268: Consolidate build scripts.

**Comment by jahorton** [comment]
Duplicate of #6516.  Closing this one in favor of that one, as that one has some diagnosis notes on it already.
